docs/openapi/resources/credential-status.yml

```diff
@@ -14,25 +14,35 @@ post:
         schema:
           type: object
           description: Request for updating the status of an issued credential.
+          required:
+            - credentialId
```

Comment:
This does't seem right. This should be just `id`, shouldn't it? 

https://www.w3.org/TR/vc-data-model/#identifiers

Reply:
Hmm. I would argue that the request body here is _referencing_ a credential by ID, not _providing_ a credential and giving its ID (which would fall under the linked vc data model). If I understand things correctly, an `id` property generally serves to identify the object in which it occurs. When interpreted that way, I think that `credentialId` is more appropriate here.

Reply:
This endpoint MUST align with https://w3c-ccg.github.io/vc-api/#update-status

Reply:
@OR13 in that case it sounds like I should modify the `type` property of each `credentialStatus` array item to be string with allowable values of "0" and "1" via `enum`, do you agree?

Reply:
yes, pretty sure that endpoint has changed since it was created.